tests/e2e_regression/damask/expected/account.body

```diff
@@ -6,5 +6,8 @@
   "debonding_delegations_balance": "0",
   "delegations_balance": "0",
   "escrow": "0",
-  "nonce": 5
+  "nonce": 5,
+  "stats": {
+    "num_txns": 0
```

Comment:
is this from transactions outside of the block scanning range? either before or after

Reply:
yeah; the txs are from before the block scanning range.